main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+	"time"
+)
+
+func main() {
+	start := time.Now()
+	var wg sync.WaitGroup
+
+	wg.Add(5)
+	go printPrimeNumbers(2, 10, &wg)
+	go printPrimeNumbers(20, 30, &wg)
+	go printPrimeNumbers(30, 40, &wg)
+	go printPrimeNumbers(50, 60, &wg)
+	go printPrimeNumbers(60, 70, &wg)
+
+	wg.Wait()
+	end := time.Now()
+	duration := end.Sub(start)
+	fmt.Println(duration)
+	fmt.Println()
+}
+
+func printPrimeNumbers(num1, num2 int, wg *sync.WaitGroup) {
```

Comment:
functions name should be appropriate & according to your functionallity like in this case PrimeNumbers()

Reply:
Done